src/app/containers/Byline/index.test.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import BylineContainer from './index';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { service as pidginServiceConfig } from '#lib/config/services/pidgin';
+
+/* eslint-disable react/prop-types */
+const BylineContainerWithContext = ({
+  pageType,
+  service,
+  serviceContext = pidginServiceConfig.default,
+  bbcOrigin = 'https://www.test.bbc.com',
+  name,
+  title,
+}) => {
+  const blocks = [{ name, title }];
+  return (
+    <ServiceContext.Provider value={serviceContext}>
+      <RequestContextProvider
+        isAmp={false}
+        pageType={pageType}
+        service={service}
+        statusCode={200}
+        bbcOrigin={bbcOrigin}
+        pathname="/pathname"
+      >
+        <BylineContainer blocks={blocks} />
+      </RequestContextProvider>
+    </ServiceContext.Provider>
+  );
+};
+
+describe('Byline', () => {
+  shouldMatchSnapshot(
+    'should render correctly for STY pages with name and title',
+    BylineContainerWithContext({
+      pageType: 'STY',
+      service: 'pidgin',
+      name: 'John Smith',
+      title: 'Art Editor',
+    }),
+  );
+
+  it('should render nothing if name and title are null', () => {
```

Comment:
A minor thing, but should "and" be "or"? It got me thinking if we should test this case by setting `name`, but not `title`. I wouldn't be surprised if there was a Person without a `title` lurking around somewhere. 😅

Reply:
Yeah should probably add the case, I'll take a look 🙂

Reply:
AFAIK title is mandatory

Reply:
Then @FK78 does it https://github.com/bbc/simorgh/pull/5512/commits/6495d52347cec1b7b47790244bc199bda67e92db 🥇